src/Domains/Validator/PublicDomain.php

```diff
@@ -70,4 +71,16 @@ public function getType(): string
     {
         return self::TYPE_STRING;
     }
+
+    /**
+     * Allow domains
+     *
+     * Add domains to the allowed domains array
+     *
+     * @param array $domains
+     */
+    public static function allow(array $domains): void
+    {
+        self::$allowedDomains = array_merge(self::$allowedDomains, $domains);
```

Comment:
```suggestion
        self::$allowedDomains[] = $domains;
```

Same logic, easier to read.